salt/config/__init__.py

```diff
@@ -1453,6 +1456,7 @@ def _gather_buffer_space():
     'grains_refresh_every': 0,
     'minion_id_caching': True,
     'minion_id_lowercase': False,
+    'minion_id_remove_domain': '',
```

Comment:
```
'minion_id_remove_domain': False
```

Reply:
ok

Reply:
Hi @DmitryKuzmenko, thank you for your suggestions. 
I never used (string, bool) before. 
Already checked in.

I will "resolve" this conversation because GitHub still sees open "requested changes".